packages/eas-cli/src/project/projectUtils.ts

```diff
@@ -128,6 +132,15 @@ export async function enforceRollBackToEmbeddedUpdateSupportAsync(
   );
 }
 
+export async function isClassicUpdatesSupportedAsync(projectDir: string): Promise<boolean> {
+  const expoUpdatesPackageVersion = await getExpoUpdatesPackageVersionIfInstalledAsync(projectDir);
+  if (expoUpdatesPackageVersion === null) {
+    return false;
+  }
+
+  return !semver.gte(expoUpdatesPackageVersion, '0.19.0');
```

Comment:
```suggestion
  return semver.lt(expoUpdatesPackageVersion, '0.19.0');
```
Nit, what do you think?

Reply:
I just realized none of this works with canary versions anyways which look like `0.0.1-canary-20240109-93608d8`.

Reply:
Will address canary stuff in follow-up PR